DataManagementSystem/Client/DataManager.py

```diff
@@ -662,6 +662,8 @@ def __replicate( self, lfn, destSE, sourceSE = '', destPath = '', localCache = '
         self.getFile( lfn, destinationDir = localDir )
         localFile = os.path.join( localDir, os.path.basename( lfn ) )
         fileDict = {destPfn:localFile}
+        #FIXME: This is to avoid third party transfer attempt by the SE, will be better treated in the future
```

Comment:
This is rather subtle. The SE metod putFile attempts to do replication rather than upload if the file size is given. In case of using the SE proxy service this results in no file download before the upload in case replication through a local cache. This is a simple fix for the case which will be treated differently in v6r13 anyway.